src/pages/Club/Club/Club.tsx

```diff
@@ -631,7 +648,7 @@ const Club = () => {
                         </Tooltip>
                       </Col>
                     ) : null}
-                    {canCreate ? (
+                    {userAccesses["DeleteClub"] ? (
```

Comment:
you mean delete or archive?

Reply:
"Delete" and "Archive" are almost the same accesses, so I decided not to split them, and leave just "Delete"

Reply:
ok